ui-v2/tests/utils/handlers.ts

```diff
@@ -1,96 +1,91 @@
 import { http, HttpResponse } from "msw";
 
+export const prefectURL = (path: string) => {
+	return `${import.meta.env.VITE_API_URL}${path}`;
```

Comment:
What if we did this?
```suggestion
	return new URL(`api/${path}`, import.meta.env.VITE_API_URL).toString();
```
I think this would work even if you accidentally left off the first `/` in `path`.

Reply:
That works, but now when its something like
`buildApiUrl('automations/')`.

The initial `/` needs to be omitted.
I've pushed up a commit if we want this pattern, but we can revert this commit

Reply:
Oh shucks, I guess it's a wash then. Either way, I think it'd be nice to be able to handle the leading `/` if it's present or not for DevEx.

Reply:
I reverted the commit to remove that change